Snowflake.Data.Tests/IntegrationTests/SFDbCommandIT.cs

```diff
@@ -175,6 +175,279 @@ public void TestExecuteAsyncWithMaxRetryReached()
                 Assert.GreaterOrEqual(stopwatch.ElapsedMilliseconds, 30 * 1000);
             }
         }
+
```

Comment:
we don't use in any test methods:
- GetResultsFromQueryId
- GetQueryStatus

It would be good to write a test which uses these methods as well.

Reply:
Those methods are used in the non-async tests of `SFDbCommandIT`. One of the tests uses [both ](https://github.com/snowflakedb/snowflake-connector-net/blob/912f4a250eb7600ce2e04f0dd235d7770afbce9a/Snowflake.Data.Tests/IntegrationTests/SFDbCommandIT.cs#L1340) [methods](https://github.com/snowflakedb/snowflake-connector-net/blob/912f4a250eb7600ce2e04f0dd235d7770afbce9a/Snowflake.Data.Tests/IntegrationTests/SFDbCommandIT.cs#L1345)